### PR TITLE
modify gitea script to mirror also private repositories

### LINKED ISSUE #31: Mirror everything
Followup of osism/issues#234: Mirror also private repositories.

### PR DESCRIPTION
**Pull request by tibeer**
closes #31.

Signed-off-by: Tim Beermann <beermann@osism.tech>

### CODE REVIEW

File: scripts/mirror.py [hunk 1]
```diff
@@ -1,16 +1,6 @@
...
-
-
-class Mirrors:
-    def __init__(self, file: str) -> None:
-        with open(file, 'r') as stream:
-            try:
-                self.list = yaml.safe_load(stream)
-            except yaml.YAMLError as exc:
-                print(exc)
+from github import Github
```
Comment (osfrickler) [inline scripts/mirror.py:3]: Please keep alphabetic ordering for imports. Also separate builtin from external modules. So this should be:

```
import os

from github import Github
import requests
```

**Review by osfrickler — COMMENTED** [review]
Being a bit more verbose in the commit message could be helpful. If I read the code correctly, all repositories in the specified org are now mirrored instead of using an explicit list? That would make a nice commit message body.

File: scripts/mirror.py (hunk 1 above)
Comment (tibeer) [reply scripts/mirror.py:3]: good hint. we should lint for that in some form. do you have a suggestion.

Comment (osfrickler) [reply scripts/mirror.py:3]: There is a check here [0] but I'm not sure how OpenStack-specific it is. We could give it a try and adapt it necessary.

[0] https://docs.openstack.org/hacking/latest/user/hacking.html#imports

**Comment by tibeer** [comment]
@osfrickler can you take a look again?

**Comment by osfrickler** [comment]
I don't see an updated commit message yet

**Review by osfrickler — APPROVED** [review]

**Pull request merged**